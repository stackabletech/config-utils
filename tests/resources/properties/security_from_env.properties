networkaddress.cache.negative.ttl=0
networkaddress.cache.ttl=foo
meine.suß.Pröpertie=42
example-password=admin-pw= withSpace$%" &&} §
