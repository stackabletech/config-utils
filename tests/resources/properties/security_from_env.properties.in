networkaddress.cache.negative.ttl=0
networkaddress.cache.ttl=${env:DISPLAY}
meine.suß.Pröpertie=42,3
