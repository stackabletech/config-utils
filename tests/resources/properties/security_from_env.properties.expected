networkaddress.cache.negative.ttl=0
networkaddress.cache.ttl=:1
meine.suß.Pröpertie=42,3
